src/Native/System.Security.Cryptography.Native/pal_eckey.cpp

```diff
@@ -12,33 +12,59 @@ extern "C" void CryptoNative_EcKeyDestroy(EC_KEY* r)
     EC_KEY_free(r);
 }
 
-extern "C" EC_KEY* CryptoNative_EcKeyCreateByCurveName(int32_t nid)
```

Comment:
Hmm ok I will keep it. I didn't know we wanted to support binary (meaning extern "c") compat in this way.